docker-compose.yml

```diff
@@ -137,7 +120,7 @@ services:
       dockerfile: rocky/Dockerfile
       args:
         ENVIRONMENT: dev
-    command: python3 manage.py runserver 0.0.0.0:80
+    command: python3 manage.py runserver 127.0.0.1:80
```

Comment:
This is an unrelated change, but also doesn't work as far as I know, because it will listen to 127.0.0.1 in the container which means you can't connect to it from outside the container?

Reply:
This change shouldn't be in here, I'll remove it.